micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/BulkUpload.js

```
@@ -68,7 +69,7 @@ const BulkUpload = ({ multiple = true, onSubmit , onSuccess }) => {
       )}
       {/* <FileUploader multiple={multiple} handleChange={handleChange} name="file" types={fileTypes} children={dragDropJSX} /> */}
       {files.length > 0 && renderFileCards}
-      {showToast && <Toast label={showToast.label} error={showToast?.isError} isDleteBtn={true} onClose={() => setShowToast(null)}></Toast>}
+      {showToast && <Toast label={showToast?.label} type={showToast?.type} isDleteBtn={true} onClose={() => setShowToast(null)}></Toast>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)